meson.build

```
@@ -506,14 +506,15 @@ if get_option('enable-test')
   )
 
   path_gst_plugin = join_paths(meson.build_root(), 'gst', 'nnstreamer')
+  path_gst_join_plugin = join_paths(meson.build_root(), 'gst', 'join')
```

How about changing like this? 
I modified other files as below. I think I missed this part. 
(i.e., line 662 at nnstreamer.spec)
```suggestion
  path_gst_plugin = join_paths(meson.build_root(), 'gst')
```